qemu/tests/blk_stream.py

```diff
@@ -29,18 +33,36 @@ def start(self):
         start block device streaming job;
         """
         params = self.parser_test_args()
-        base_image = params.get("base_image")
         default_speed = params.get("default_speed")
 
         error.context("start to stream block device", logging.info)
-        self.vm.block_stream(self.device, default_speed, base_image)
+        self.vm.block_stream(self.device, default_speed, self.base_image,
+                             backing_file=self.backingfile)
         status = self.get_status()
         if not status:
             raise error.TestFail("no active job found")
         msg = "block stream job running, "
         msg += "with limited speed %s B/s" % default_speed
         logging.info(msg)
 
+    def get_base_backingfile(self):
+        """
+        Get values for base and backing-file option.
+        """
+        params = self.parser_test_args()
+        backingfile_option = params.get("backingfile_option", "no")
+        if backingfile_option == "no":
+            return
+
+        self.base_image = self.get_image_file()
+        absolute_path = params["absolute_path"]
+        if absolute_path == "yes":
+            self.backingfile = self.base_image
+        else:
+            current_path = process.system_output("pwd")
```

Comment:
@vivianQizhu 
Seems the output of pwd is:
15:56:58 INFO | Running 'pwd'
15:56:58 DEBUG| [stdout] /home/staf-kvm-devel

Seems it is not the pwd of the image, is it expected?